server/command.go

```diff
@@ -29,12 +31,30 @@ func getCommand(client *pluginapi.Client) *model.Command {
 		DisplayName:          "MS Teams Meetings",
 		Description:          "Integration with MS Teams Meetings.",
 		AutoComplete:         true,
-		AutoCompleteDesc:     "Available commands: start, disconnect",
+		AutoCompleteDesc:     "Available commands: help, start, disconnect",
```

Comment:
We should put `help` last here too

```suggestion
		AutoCompleteDesc:     "Available commands: help, start, disconnect",
```